### PR TITLE
Add hungarian language option and dictionary

### PR DESCRIPTION
**Pull request by goplayoutside3**
Staging branch URL: https://pr-7169.pfe-preview.zooniverse.org
Example project url: https://pr-7169.pfe-preview.zooniverse.org/projects/benjamin-dot-richards/oceaneyes?language=hu

Follows https://github.com/zooniverse/Panoptes-Front-End/pull/7140

Add hungarian language option and dictionary

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `npm ci` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on GitHub Actions?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

File: app/constants/languageMenu.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ export default {
   he: 'עברית',
   hi: 'हिन्दी',
   hr: 'Hrvatski',
+  hu: 'magyar nyelv',
```
Comment (lcjohnso) [inline app/constants/languageMenu.js:14]:
```suggestion
  hu: 'Magyar',
```

**Review by shaunanoordin — APPROVED** [review]
## PR Review

This PR adds Hungarian language support to PFE. Testing on staging with macOS + Chrome checks out OK. LGTM!

File: app/constants/languageMenu.js (hunk 1 above)
Comment (lcjohnso) [reply app/constants/languageMenu.js:14]: Following example from Wikipedia’s language selector (https://www.wikipedia.org/) and others (e.g. https://confluence.atlassian.com/proforma/internationalization-1087521479.html)

**Review by lcjohnso — APPROVED** [review]
One tweak to the language label, but otherwise ready to merge!

**Pull request merged**